### DISCUSSION TITLE
Dimness for wrappers ()[]{} and misc background elements

### DISCUSSION THREAD

**Discussion by dominicva** [BeardedBear/bearded-theme#81] [Q&A] [answered]
Hi there ☺️☺️

First of all – **congrats!** 

I've browsed all the themes... saying this is the best would be cheesy, but I definitely haven't seen one I prefer!

**I have one issue:** some character types are so dim I struggle to see them (tbf my vision isn't perfect).
Check out the screenshot. It's things like the enclosing `[], {}, ()` and comments too tbh. 

Is this just me?

🙏🙏

<img width="704" alt="Screenshot 2022-05-24 at 19 22 59" src="https://user-images.githubusercontent.com/26066449/170153814-5a390147-98e9-4c18-948b-87b57a91ab88.png">

**Comment by BeardedBear** [comment, marked as answer]
I recently created a new series (HC) that solves this problem, which focuses on a better contrast, and therefore a better readability, especially on the parentheses & co :)
